src/tga_file.h

```diff
@@ -0,0 +1,60 @@
+
+
+#ifndef __TGA_FILE_H_INCLUDED__
+#define __TGA_FILE_H_INCLUDED__
+
+#include<iostream>
+#include<vector>
+#include<fstream>
+
+using namespace std;
+typedef char byte;
+
+class WriteTGA {
+public: 
+
+	byte header[18] = {
+		0,                  //1      id
+		0,                  //2      Color map type
+		0,                  //3      24bit uncompressed RGB
+		0, 0, 0, 0,		    //4-7    Color map related data
+		0, 0,               //8-9    X origin
+		0, 0,               //10-11  Y origin
+		0, 0,               //12-13  Width
+		0, 0,               //14-15  Height
+		24,                 //16     Means 24bit RGB
+		0                   //17     Image descriptor
+	};
+	struct rgb {
+		byte red, green, blue;
+	};
+	void writeTargaFile(const char* fileName, int height, int width) {
+
+		ofstream file;
+		rgb* data = new rgb;
+
+		file.open(fileName, ios::binary|ios::out);
+		if (!file.is_open()) cout << "file is not open!";
```

Comment:
1. If this happens it makes sense to throw an exception and quit.
1. Please avoid one-line `if`-statements. They look short but they are less readable that classic  
   
   ```
   if () {
     // some code
   }
   ```